cumulusci/tasks/metadata_etl/field_sets.py

```diff
@@ -0,0 +1,70 @@
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_bool_arg, process_list_arg
+
+
+class AddFieldsToFieldSet(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "field_set": {
+            "description": "Name of field set to affect, in Object__c.FieldSetName form.",
+            "required": True,
+        },
+        "fields": {
+            "description": "Array of field API names to add to the field set. "
+            "Can include related fields using AccountId.Name or Lookup__r.CustomField__c style syntax.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
+        super()._init_options(kwargs)
+        self.api_names = set(
+            [self._inject_namespace(self.options["field_set"].split(".")[0])]
+        )
+
+        if "fields" not in self.options:
```

Comment:
I might do

```suggestion
        if not self.options.get("fields"):
```

so that we also show an error if the option is set as an empty list.